### PR TITLE
Compose per integration metadata.yaml and ops_agent_metadata.yaml files against the ops-agent repo 2.32.0 branch.

### PR DESCRIPTION
**Pull request by stackdriver-instrumentation-release**

### CODE REVIEW

**Comment by mtabasko** [comment]
This PR was generated by the tool that generates the Ops Agent third-party docs, which I just ran -- wanted to pick up new metrics in 2.32.0.

File: integrations/elasticsearch/ops_agent_metadata.yaml [hunk 1]
```diff
@@ -298,3 +298,24 @@ platforms:
     kind: GAUGE
     labels:
     - state
+  - name: workload.googleapis.com/elasticsearch.cluster.in_flight_fetch
```
Comment (varun-c) [inline integrations/elasticsearch/ops_agent_metadata.yaml:301]: This should also result in an update to the metrics_minimum_supported version as these default metrics will not be collected as part of the current version

File: integrations/mysql/ops_agent_metadata.yaml [hunk 2]
```diff
@@ -212,6 +212,14 @@ platforms:
     kind: CUMULATIVE
     labels:
     - operation
+  - name: workload.googleapis.com/mysql.replica.time_behind_source
```
Comment (varun-c) [inline integrations/mysql/ops_agent_metadata.yaml:215]: Same comment applies to this too.

File: integrations/elasticsearch/ops_agent_metadata.yaml (hunk 1 above)
Comment (sophieyfang) [reply integrations/elasticsearch/ops_agent_metadata.yaml:301]: https://github.com/GoogleCloudPlatform/ops-agent/pull/1279 shall update the version, after the PR merged in, we will rerun the script to re-generate this content PR

**Comment by EvanSimpson** [comment]
Resolved in #602

File: integrations/elasticsearch/ops_agent_metadata.yaml (hunk 1 above)
Comment (igorpeshansky) [reply integrations/elasticsearch/ops_agent_metadata.yaml:301]: These metrics were apparently already collected since 2.21.0, but undocumented.

**Pull request closed without merging**